.browserslistrc

```diff
@@ -1,7 +1,11 @@
-[production staging]
-> 1%
-ie 10
+# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+#
+# For IE 10-11 support, please remove 'not' from the last line of the file and adjust as needed
 
-[development]
-last 1 chrome version
-last 1 firefox version
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
```

Comment:
i don't find the coverage of not dead: https://browserl.ist/?q=not+dead
instead i can see the others, eg. https://browserl.ist/?q=%3E+0.5%25